lua/competitest/init.lua

```diff
@@ -46,6 +46,7 @@ local default_config = {
 			view_output = { "a", "A" },
 			view_stdout = { "o", "O" },
 			view_stderr = { "e", "E" },
+			toggle_diff = { "D", "<C-d>" },
```

Comment:
You're right it should be `d`. I was getting delay with `d` that's why I had chosen `D`. Turns out it's because of other plugin I'm using. I'll change this.